rules/android/android_library.bzl

```
@@ -86,3 +87,11 @@ def android_library(
         name = name + ".lint",
         target = name,
     )
+
+    android_lint_test(
+        name = name + "_lint_test",
+        srcs = attrs.get("srcs", default = []),
+        resource_files = resources_xml(resource_files),
+        lib = name,
+        deps = android_library_deps,
```

Passing the manifest into lint. similar to the comment in `android_binary`

```suggestion
        deps = android_library_deps,
        manifest = attrs.get("manifest", default = None),
```